static/app/components/asyncComponentSearchInput.tsx

```diff
@@ -15,7 +16,28 @@ type RenderProps = {
   value: string;
 };
 
-type DefaultProps = {
+function DefaultSearchBar({
+  busy,
+  onSubmit,
+  className,
+  placeholder,
+  handleInputChange,
+  query,
+}): React.ReactElement {
+  return (
+    <Form onSubmit={onSubmit}>
+      <Input
+        value={query}
+        onChange={handleInputChange}
+        className={className}
+        placeholder={placeholder}
+      />
+      {busy ? <StyledLoadingIndicator size={18} hideMessage mini /> : null}
+    </Form>
+  );
+}
+
+export interface AsyncComponentSearchInputProps extends WithRouterProps {
```

Comment:
Should we be starting to use interfaces 😬

I actually forget what we landed on in the TSC

Reply:
I think because we are extending the props here, that would be the correct use. If it was a flat type, either would be ok imo